pkg/v3/plugin/ocr3.go

```diff
@@ -66,8 +68,50 @@ func (plugin *ocr3Plugin[RI]) ValidateObservation(outctx ocr3types.OutcomeContex
 	panic("ocr3 ValidateObservation not implemented")
 }
 
-func (plugin *ocr3Plugin[RI]) Outcome(outctx ocr3types.OutcomeContext, query types.Query, aos []types.AttributedObservation) (ocr3types.Outcome, error) {
-	panic("ocr3 Outcome not implemented")
+func (plugin *ocr3Plugin[RI]) Outcome(outctx ocr3types.OutcomeContext, query types.Query, attributedObservations []types.AttributedObservation) (ocr3types.Outcome, error) {
+	type resultAndCount struct {
+		result ocr2keepers.CheckResult
+		count  int
+	}
+
+	resultCount := make(map[string]resultAndCount)
+
+	for _, attributedObservation := range attributedObservations {
+		observation, err := ocr2keepersv3.DecodeAutomationObservation(attributedObservation.Observation)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, result := range observation.Performable {
+			uid := fmt.Sprintf("%v", result.Payload)
```

Comment:
Ideally the entire `CheckResult` should be identical.

Reply:
I wasn't sure if retryable would vary across different nodes, as in, for one node the check was successful the first time, but on another node the same check failed initially but was successful afterwards?

Reply:
Retryable is reset for every run through the check pipeline. In fact, the check pipeline only receives an UpkeepPayload so it doesn't know anything about a previous run. I think the only deviation between these check results is in the specific values of fastgas and linknative prices. But we rely on all of it to be identical anyway in the OCR2 version and it's working fine there.